package.json

```diff
@@ -41,6 +41,7 @@
     "webpack-dev-server": "^4.8.1"
   },
   "dependencies": {
+    "azure-devops-extension-sdk": "^4.0.2",
```

Comment:
Does this need to be a dependency or could it work also if it is a dev dependency